pkg/metrics/builder.go

```diff
@@ -44,20 +50,24 @@ type Builder struct {
 	handler http.Handler
 }
 
-// Bind defines command line flags and binds their values to the builder fields.
-func (b *Builder) Bind(flagSet *flag.FlagSet) {
-	flagSet.StringVar(
-		&b.Backend,
-		"metrics-backend",
+// AddFlags adds flags for Builder.
+func AddFlags(flags *flag.FlagSet) {
+	flags.String(
+		metricsBackend,
 		"expvar",
```

Comment:
nit: this should probably be a `defaultMetricsBackend` const, and same for the route